api-catalog-ui/frontend/src/components/Wizard/WizardInputs.jsx

```diff
@@ -106,13 +107,6 @@ class WizardInputs extends Component {
         return this.renderInputs(dataAsObject.content, 1);
     };
 
-    loadButtons() {
```

Comment:
Why have you reverted this small refactoring?

Reply:
It was supposed to house another button, but we decided against it - creating a function for a single button seemed excessive :)

Reply:
Alright, got it :)